integration-tests/eks/validateResources_test.go

```diff
@@ -26,8 +26,11 @@ const (
 	nameSpace        = "amazon-cloudwatch"
 	addOnName        = "amazon-cloudwatch-observability"
 	agentName        = "cloudwatch-agent"
-	podNameRegex     = "(" + agentName + "|" + addOnName + "-controller-manager|fluent-bit)-*"
-	serviceNameRegex = agentName + "(-headless|-monitoring)?|" + addOnName + "-webhook-service"
+	operatorName     = addOnName + "-controller-manager"
+	fluentBitName    = "fluent-bit"
+	dcgmExporterName = "dcgm-exporter"
+	podNameRegex     = "(" + agentName + "|" + operatorName + "|" + fluentBitName + ")-*"
```

Comment:
Shouldnt `podNameRegex` also be updated to reflect the dcgm exporter pods?

Reply:
no, since there will be no dcgm pod with the current integ test setup.